src/routes/(site)/events/event.svelte

```diff
@@ -108,7 +99,7 @@
             info.acmPath.slug
           )}
       >
-        <CopyText />
+        <img src="/assets/svg/copy-text.svg" alt="copy text button" />
```

Comment:
![image](https://user-images.githubusercontent.com/8981287/197419563-7890408b-8bff-4f67-a9e8-6f2446fd91cd.png)

Reply:
> copy button not respond to dark mode
> ![image](https://user-images.githubusercontent.com/8981287/197419563-7890408b-8bff-4f67-a9e8-6f2446fd91cd.png)

yeah I just realized that now, probably make an <img> element instead of using background-image

Reply:
done